website/docs/en-US/popover.md

```diff
@@ -34,20 +36,23 @@ Similar to Tooltip, Popover is also built with `Vue-popper`. So for some duplica
     ref="popover"
     placement="right"
     title="Title"
-    width="200"
+    :width="200"
     trigger="focus"
-    content="this is content, this is content, this is content">
+    content="this is content, this is content, this is content"
+  >
+    <template #reference>
+      <el-button>Focus to activate</el-button>
+    </template>
   </el-popover>
- 
-  <el-button v-popover:popover>Focus to activate</el-button>
 
   <el-popover
     placement="bottom"
     title="Title"
-    width="200"
+    :width="200"
     trigger="manual"
     content="this is content, this is content, this is content"
-    v-model:visible="visible">
+    :visible="visible"
```

Comment:
We need to use `v-model:visible` simply binding the value won't get updated after `@click` event occurs